SICP_book/chapter_1_3.metta

```diff
@@ -0,0 +1,485 @@
+;needed for putting lambda functions as input to functions
+(: Lambda (-> $a $t))
+(: lambda (-> Variable Atom Lambda))
+(= ((lambda $var $body) $val)
+    (let $var $val $body) )
+
+; for lambda without any input
+(= (lambda0 $body)
```

Comment:
It would be more aligned with other definitions of lambda if you define `lambda0` as `(= ((lambda0 $body)) $body)` and then all usages should be wrapped by parenthesis: `($func-h)`. The type of `lambda0` is `(: lambda0 (-> Atom (-> $t)))`